content/source/docs/cloud/api/variable-sets.html.md

```diff
@@ -0,0 +1,613 @@
+| `:varset_id`         | The variable set ID              |
+| `:var_id`            | The ID of the variable to delete |
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "type": "vars",
+    "attributes": {
+      "key": "g6e45ae7564a17e81ef62fd1c7fa86138",
+      "value": "61e400d5ccffb3782f215344481e6c82",
+      "description": "new cheeeese",
+      "sensitive": false,
+      "category": "terraform",
+      "hcl": false
+    }
+  }
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request PATCH \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/varsets/varset-4q8f7H0NHG733bBH/relationships/vars/var-EavQ1LztoRTQHSNT
+```
+
+### Sample Respone
+
+```json
+{
+  "data": {
+    "id":"var-EavQ1LztoRTQHSNT"
+    "type": "vars",
+    "attributes": {
+      "key": "g6e45ae7564a17e81ef62fd1c7fa86138",
+      "value": "61e400d5ccffb3782f215344481e6c82",
+      "description": "new cheeeese",
+      "sensitive": false,
+      "category": "terraform",
+      "hcl": false
+    }
+  }
+}
+```
+
+## Delete a Variable in a Variable Set
+
+`DELETEvarsets/:varset_id/relationships/vars/:var_id`
+
+| Parameter            | Description                      |
+| -------------------- | ---------------------------------|
+| `:varset_id`         | The variable set ID              |
+| `:var_id`            | The ID of the variable to delete |
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request DELETE\
+  --data @payload.json \
+  https://app.terraform.io/api/v2/varsets/varset-4q8f7H0NHG733bBH/relationships/vars/var-EavQ1LztoRTQHSNT
+```
+
+on success, responds with no content
+
+## List Variables in a Variable Set
+
+`GET varsets/:varset_id/relationships/vars`
+
+| Parameter            | Description         |
+| -------------------- | --------------------|
+| `:varset_id`         | The variable set ID |
+
+### Sample Response
+
+```json
+{
+  "data": [
+    {
+      "id": "var-134r1k34nj5kjn",
+      "type": "vars",
+      "attributes": {
+        "key": "F115037558b045dd82da40b089e5db745",
+        "value": "1754288480dfd3060e2c37890422905f",
+        "sensitive": false,
+        "category": "terraform",
+        "hcl": false,
+        "created-at": "2021-10-29T18:54:29.379Z",
+        "description": nil
+      },
+      "relationships": {
+        "varset": {
+          "data": {
+            "id": "varset-992UMULdeDuebi1x",
+            "type": "varsets"
+          },
+          "links": { "related": "/api/v2/varsets/1" }
+        }
+      },
+      "links": { "self": "/api/v2/vars/var-BEPU9NjPVCiCfrXj" }
+    },
+  ],
+  "links": {
+    "self": "<page URL>",
+    "first": "<page URL>",
+    "prev": nil,
+    "next": nil,
+    "last": "<page URL>"
+  }
+}
+```
+
+### Workspaces
+
+## Assign Variable Set to Workspaces
```

Comment:
In the docs, we're referring to this as "applying" the variable set, rather than assigning it
```suggestion
## Apply Variable Set to Workspaces
```